core/jobs/batch_jobs/exp_version_history_computation_job.py

```diff
@@ -65,26 +69,53 @@ class FormattedModelGroupDict(TypedDict):
     )
 
 
+class UnformattedModelGroupForComputationJobDict(TypedDict):
+    """Dictionary representing an unformatted model group for the computation
+    job.
```

Comment:
Ditto: Unclear what "the computation job" refers to.

Reply:
Ditto below.

Reply:
Updated it.